apps/condo/domains/organization/integrations/sbbol/sync/syncPaymentRequestsForSubscriptions.js

```diff
@@ -1,54 +1,71 @@
 const { ServiceSubscription, ServiceSubscriptionPayment } = require('@condo/domains/subscription/utils/serverSchema')
 const { getSchemaCtx } = require('@core/keystone/schema')
 const dayjs = require('dayjs')
-const { SbbolRequestApi } = require('../SbbolRequestApi')
-const { SbbolFintechApi } = require('../SbbolFintechApi')
+const { initSbbolFintechApi } = require('../SbbolFintechApi')
 const { SBBOL_YEARLY_SUBSCRIPTION_PRICE, SUBSCRIPTION_TYPE, SUBSCRIPTION_PAYMENT_STATUS, SUBSCRIPTION_PAYMENT_CURRENCY } = require('@condo/domains/subscription/constants')
+
 const conf = process.env
-const SBBOL_CONFIG = conf.SBBOL_CONFIG ? JSON.parse(conf.SBBOL_CONFIG) : {}
-const { debugMessage } = require('../common')
+const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
+
+const { logger: baseLogger } = require('../common')
 const { dvSenderFields, BANK_OPERATION_CODE } = require('../constants')
 const { SBBOL_OFFER_ACCEPT_IS_NULL_ERROR, SBBOL_OFFER_ACCEPT_HAS_INCORRECT_PAYER_REQUISITES_ERROR } = require('@condo/domains/subscription/constants/errors')
 
+const logger = baseLogger.child({ module: 'syncPaymentRequestsForSubscriptions' })
```

Comment:
somewhere you use a new logger like for `sbbol/auth` here you use child. Try to be consistent

Reply:
Changed and made consistent in `26ea5928`